install.sh

```diff
@@ -144,7 +145,7 @@ if [ "$SKIP_MATRIX" = false ]; then
     # Checkout the branch or commit specified for rpi-rgb-led-matrix
     git checkout $DRIVER_SHA
     git pull
-    make build-python PYTHON="$PYTHON"
+    make build-python PYTHON="$PYTHON" CYTHON=cython
```

Comment:
on my system, the `cython3` apt package installs under `cython3`, not `cython`. Pip, on the other hand, installs under `cython`

Reply:
Cool, was shooting in the dark til I could find a machine to test on but I got this working with `cython3`